_posts/en/2020-04-21-how-is-the-public-service-managing-large-IT-project.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "How is the Public Service Managing Large IT Projects?"
+ref: how-is-the-public-service-managing-large-IT-project.md
+lang: en
+author: My Thuy Pham-Vo on behalf of the"IT Strategy Team"
+date: "2020-04-21"
+last_modified: "2020-04-21"
+excerpt_separator: <!--more-->
+---
+
+Having large projects raises concerns on the budget, available resources, the estimated success and the risks. Even if many kinds of research have informed the success of small projects are higher than large projects, many government departments and agencies still decide to proceed to implement large IT projects. As the Government of Canada works to improve the use of information technology in its operations, we have reviewed reports from the Office of the Audit General of Canada (OAG) on IT projects undertaken. 
+ 
+## Adapting to Priorities
+Looking back at the [2006 Audit Report](https://www.oag-bvg.gc.ca/internet/English/parl_oag_200611_03_e_14971.html), the Office of the Audit General of Canada (OAG) has assessed a sample of IT projects provided by the Treasury Board Secretariat (TBS). The issues raised in the previous audits were inadequate analysis of underlying business issues and support from senior executives and project ownership. OAG highlights for all IT projects to support business transformation and operational needs. All project submissions must also consider IT components. The following Audit reports note resembling challenges in delivering large IT projects. 
+
+First, the attention and support of senior management must improve internal reviews of IT projects, including the evaluation and analysis of key decision documents. Nearly all the reports acknowledge the need to strengthen the project governance and accountabilities. The reports also state that projects were approved even if the proposals did not meet the prerequisite requirements. OAG recommends to improve the mechanism and find the best practice to maximize the capabilities and commitments for the overall direction of the projects.
+
+The condition of the IT systems and infrastructure, "Aging technology", draws attention in 2010. Aging IT Systems like the Personal Income Tax and Benefits Return administration system and the Employment Insurance program was noticed in the Chief Information Officer Branch (CIOB) of all the departments and agencies but did not have any strategic directions. OAG highly recommends all organizations to develop a strategic approach in the IT investments and risk management to approach the significant actions in the health of the IT systems.
```

Comment:
Add another sentence:

"This Spring Auditor General Report lead to formalizing the approach to Application Portfolio Management (APM) across the federal departments and the creation of a TBS requirement for departments to report on health of their critical infrastructure"

link to APM program from TBS (internal to GC as it's on GCPedia): https://www.gcpedia.gc.ca/wiki/OCIO_Application_Portfolio_Management